rbuild.ini

```diff
@@ -19,6 +19,7 @@ define =
 [develop]
 cxx = ${rocm_path}/llvm/bin/clang++
 cc = ${rocm_path}/llvm/bin/clang
+ignore = ROCmSoftwarePlatform/rocMLIR
```

Comment:
Why does this one ignore it ? I use `rbuild develop` to install dependencies.

Reply:
Yea this is the wrong place. It should be under the `gh` session.